scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -31,8 +34,11 @@ remove_dotnet_pkgs
 [ "$?" -ne 0 ] && echo "Failed to remove dotnet packages." && exit 1
 
 echo "Deleting install root - $dotnet_install_root"
-rm -rf $dotnet_install_location
+rm -rf $dotnet_install_root
```

Comment:
Thanks for fixing this. I renamed some variables as I made addressed some PR feedback and it did not test it enough :(